### PR TITLE
Simple example with tests

### PR DESCRIPTION
**Pull request by isnifer**
@deepsweet I want to discuss this simple example of component and tests for it. What do you think?

### CODE REVIEW

File: examples/simple/src/Simple.es6 [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                validators={this.props.validators || this.validators()}>
+                <div>
+                    <input type="text" value={this.state.value} onChange={::this._onInputChange} />
+                </div>
+            </Validator>
+        );
+    }
+}
+
+if (!global) {
```
Comment (deepsweet) [inline examples/simple/src/Simple.es6]: :open_mouth: do we really need this?

File: examples/simple/src/Simple.es6 [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import Valya from '../../../build/index';
```
Comment (deepsweet) [inline examples/simple/src/Simple.es6:2]: I'm not sure about how to test this example live, maybe we need to create separate repo with webpack and other env stuff?

Comment (mistadikay) [inline examples/simple/src/Simple.es6:2]: maybe it's better to just go with the `examples` folder in _this_ repo

File: examples/simple/src/Simple.es6 (hunk 1 above)
Comment (isnifer) [inline examples/simple/src/Simple.es6]: Sorry for delay, I was in Egypt.
Of course no, I can create separate file for browser, which will be import this component and render to mountNode

File: examples/simple/src/Simple.es6 (hunk 2 above)
Comment (isnifer) [inline examples/simple/src/Simple.es6:2]: I think about separate repo and with these thoughts I started this PR to discuss with you, guys.

**Pull request closed without merging**